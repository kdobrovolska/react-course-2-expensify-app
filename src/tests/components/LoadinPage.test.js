import React from 'react'
import {shallow} from 'enzyme'
import LoadingPage from '../../components/LoadingPage';

test('LoadingPage should be rendred properly',()=>{
    const wrapper= shallow(<LoadingPage/>);
    expect(wrapper).toMatchSnapshot();  
});