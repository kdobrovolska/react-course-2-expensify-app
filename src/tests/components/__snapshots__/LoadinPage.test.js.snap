// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should be rendred properly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__img"
    src="loader.gif"
  />
</div>
`;
